packages/edit-site/src/components/global-styles/variations/variation.js

```diff
@@ -15,12 +15,18 @@ import { privateApis as blockEditorPrivateApis } from '@wordpress/block-editor';
  * Internal dependencies
  */
 import { mergeBaseAndUserConfigs } from '../global-styles-provider';
+import getValueFromObjectPath from '../../../utils/get-value-from-object-path';
+import setNestedValue from '../../../utils/set-nested-value';
 import { unlock } from '../../../lock-unlock';
 
 const { GlobalStylesContext, areGlobalStyleConfigsEqual } = unlock(
 	blockEditorPrivateApis
 );
 
+const PATHS_TO_AVOID_OVERWRITING = [
```

Comment:
```suggestion
const PATH_TO_AVOID_OVERWRITING = [
```
I think this should just say "Path" as it's only one path not multiple.

Reply:
Surely it's an array of paths. Currently it's one but it could be more in future.

Reply:
Oh yeah I missed that it was a nested array.